wideq/core.py

```diff
@@ -137,10 +152,16 @@ def __init__(self, code, message):
 class NotLoggedInError(APIError):
     """The session is not valid or expired."""
 
+    def __init__(self):
```

Comment:
I think we should not add these empty-argument constructors—it's actually good to keep around the error code and message!